.github/CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# these owners will be requested for review when someone
+# opens a pull request.
+*       @dav3r @felddy @jsf9k @mcdonnnj @cisagov/team-ois
```

Comment:
I know this will update with a Lineage PR, but since you added it I wasn't sure if you wanted to include @hillaryj for completeness.

Reply:
Good catch- not sure where I managed to grab an older CODEOWNERS file from, but fixed with https://github.com/cisagov/pca-gophish-composition-packer/pull/5/commits/e09cefd1e538cce9407055e4398076de02db6066 - thanks!